extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -86,7 +86,7 @@ export class AmpAnalytics extends AMP.BaseElement {
     this.type_ = null;
 
     /** @private {!boolean} */
-    this.isSandbox_ = element.hasAttribute('sandbox');
+    this.isSandbox_ = false;
```

Comment:
Determine if we want to scope selector inside parentElement, support all url variables and so...
I agree it's safer to initialize it to true. But it won't be used after `buildCallback`, so I think it does not matter so much.